software/docusaurus.yml

```
@@ -0,0 +1,15 @@
+name: "docusaurus"
+website_url: "https://docusaurus.io/"
+source_code_url: "https://github.com/facebook/docusaurus"
+description: "Docusaurus is a React-based static-site generator that creates interactive single-page applications, ideal for versatile websites like documentation, blogs, personal sites, products, and marketing pages."
+licenses:
+  - MIT
+platforms:
+  - Javascript
+  - Typescript
+  - React
+tags:
+  - Wikis
+  - Static Site Generators
```

As this is more of an SSG rather than a wiki, this should be removed as it is nit natively a wiki project,].

```suggestion
tags:
  - Static Site Generators
```